src/wazuh.py

```diff
@@ -371,6 +373,22 @@ def _generate_cluster_snippet(
     """
 
 
+def _generate_syslog_snippet() -> str:
+    """Generate the snippet for syslog configuration.
+
+    Returns: the content for the remote node for the Wazuh configuration.
+    """
+    return """
+        <remote>
+            <connection>syslog</connection>
+            <port>514</port>
+            <protocol>tcp,udp</protocol>
+            <allowed-ips>0.0.0.0/0</allowed-ips>
+            <local_ip>0.0.0.0</local_ip>
+        </remote>
+    """
```

Comment:
Out of curiosity, do we need to do `textwrap.dedent()`?

Reply:
Since the integration test passes, I'm assuming not? ;)

Reply:
Indentation is not relevant ;) The resulting file is easier to read this way